console/console-init/ui/src/components/AdapterList/AdapterList.tsx

```diff
@@ -35,23 +31,33 @@ export const AdapterList: React.FC<IAdapterListProps> = ({ id, adapters }) => {
   return (
     <div id={id}>
       {adapters &&
-        adapters.map((adapter: IAdapter, index: number) => (
-          <Adapter adapter={adapter} key={`adapter-${adapter.type}`} />
+        adapters.map((adapter: IAdapter) => (
+          <>
```

Comment:
Use `React.Fragment` and pass a key to the fragment as well. Using `<></>` inside map callback gives rise to warnings while testing.

Reply:
done